.github/workflows/build-testbenches.yml

```diff
@@ -0,0 +1,61 @@
+name: Build testbenches
```

Comment:
Rename the filename from `.github/workflows/build-testbenches.yml` to `.github/workflows/build-Artifact.yml`, which implies all benchmarks and test programs.

Reply:
Use the uppercase A for `build-Artifact`?

Reply:
> Use the uppercase A for `build-Artifact`?

No, simply name it `build-artifact`.